source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -354,3 +352,187 @@ to restart the MinIO service:
 
 Once the deployment is online, use :mc:`mc admin info` to confirm the
 uptime of all remaining servers in the deployment.
+
+.. _minio-decommission-multiple-pools:
+
+Decommission Multiple Server Pools
+----------------------------------
+
+.. versionchanged:: RELEASE.2023-01-18T04-36-38Z
+
+You can decommission multiple server pools from the same command.
```

Comment:
Same thought w.r.t. "Simultaneously" vs the focus being command syntax.

Reply:
Again, draining does _not_ happen at the same time, so I'm avoiding `simultaneously`.

Reply:
The issue here is that the change is a syntax change. Now we can list multiple pools while issuing the command. All pools lock down writes immediately, but the draining happens one pool at a time, in the order of pool listing in the command.

Reply:
> All pools lock down writes immediately

OK, so if this is the case, I think 365-367 needs some slight tidying.

- We state MinIO stops write access to all decommissioning pools
- But decommissioning happens one pool at a time
- So given 4 pools and 3 queued, how many pools can process writes?

It's either 1 pool (the one actively decommissioning) or 3 pools (any pool marked for decommissioning, whether in progress or queued).

Lets clarify that and sharpen this graf.